src/webextension/locales/fr_FR/list.ftl

```diff
@@ -0,0 +1,134 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## common
+
+document =
+  .title = Enregistrements Lockbox
+
+all-items-get-started =
+  Lorsce que vous ajoutez un compte, Il est automatiquement affiché ici.
+
+all-items-no-results = Aucun résultats
+
+item-fields-title = Nom
+item-fields-title-input
+  .placeholder = p. ex. banque primaire
+item-fields-origin = Adresse du site web
+item-fields-origin-input
+  .placeholder = www.exemple.fr
+item-fields-username = Nom d'utilisateur
+item-fields-username-input
+  .placeholder = nom@exemple.fr
+item-fields-copy-username
+  .title = Copier le nom d'utilisateur dans le presse-papier
+item-fields-password = Mot de passe
+item-fields-copy-password
+  .title = Copier le mot de passe dans le presse-papier
+item-fields-notes = Notes
+
+item-summary-new-title = Nouvel enregistrement
+item-summary-title =
+  { $length ->
+     [0]     (no site name)
```

Comment:
The `(no site name)` string should probably be translated here.

Reply:
Fixed